Makefile

```diff
@@ -5,23 +5,26 @@ test: all
 
 testall: backendtest infrastructuretest simple-haskell-marker-test
 
-backend:
+backend: repo
 	cd backend && mvn -T 1C package -q -Dmaven.test.skip=true
 
-backendtest:
+backendtest: backend
 	cd backend && mvn -T 1C test -q
 
-infrastructure: simple-haskell-marker
+infrastructure: repo
```

Comment:
infrastructure DOES need simple-haskell-marker, as the microservice updater is linked in directly